public/app/plugins/datasource/cloud-monitoring/components/SLO/LookbackPeriodSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
We are in the process of migrating the components to the new UI. Can you create a similar component in public/app/plugins/datasource/cloud-monitoring/components/Experimental/LookbackPeriodSelect.tsx? You can take public/app/plugins/datasource/cloud-monitoring/components/Experimental/Selector.tsx as an example of the new way of writing components.

Also, if you want to test the new components, set:

```
[feature_toggles]
cloudMonitoringExperimentalUI = true
```

in your configuration (INI) file (e.g. conf/custom.ini)

Reply:
Sure, added and tested https://github.com/grafana/grafana/pull/53710/commits/3ec546c0cdd5b4af03d4cbc4a1cc12d754305b23
![image](https://user-images.githubusercontent.com/8341422/185507948-e57df886-357f-48fe-8009-e72a6faca50e.png)